Mathlib/CategoryTheory/Preadditive/Injective.lean

```diff
@@ -1,9 +1,10 @@
 /-
 Copyright (c) 2022 Jujian Zhang. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Jujian Zhang, Kevin Buzzard
+Authors: Jujian Zhang, Kevin Buzzard, Joël Riou
 -/
 import Mathlib.CategoryTheory.Preadditive.Projective
+import Mathlib.CategoryTheory.MorphismProperty.LiftingProperty
```

Comment:
Would it be inconvenient to rename `Injective.lean` to `Injective/Basic.lean` and put the new results into `Injective/LiftingProperties.lean`?